ui/app/components/auth-jwt.js

```diff
@@ -131,7 +132,7 @@ export default Component.extend({
     // The namespace can be either be passed as a query paramter, or be embedded
     // in the state param in the format `<state_id>,ns=<namespace>`. So if
     // `namespace` is empty, check for namespace in state as well.
-    if (namespace === '') {
+    if (namespace === '' || this.featureFlagService.managedNamespaceRoot) {
```

Comment:
see [here](https://github.com/hashicorp/vault/blob/master/ui/app/services/feature-flag.js#L17) in code for what this returns.